app/mailers/custom_devise_mailer.rb

```diff
@@ -7,4 +8,8 @@ def confirmation_instructions(record, token, options = {})
     @confirmation_period = user_decorator.confirmation_period
     super
   end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
+  end
```

Comment:
@brendansudol while in general I am in favor of avoiding images in email, I think there is some effort being put into consistent branding to help with security UX.